src/content/community/releases/_index.en.md

```diff
@@ -5,6 +5,56 @@ weight = 40
 +++
 <!-- markdownlint-disable no-duplicate-header -->
 
+## v0.15.0
+
+### New features
+
+* To be compliant with the [Kubernetes Multicluster Services specification][MCS KEP], service discovery now distributes a single aggregated
+  ServiceImport to each cluster in the exported service's namespace. Previously each cluster distributed its own ServiceImport copy that was
+  placed in the `submariner-operator` namespace.
+* Submariner can now be installed on a dual-stack Kubernetes cluster although, currently, only IPv4 addresses are supported.
+* Add support for selecting the node to schedule `submariner-operator`, `submariner-lighthouse-agent`, `submariner-lighthouse-coredns`, and
```

Comment:
@vthapar to review this. Maybe should not be a release note until we have subctl support?

Reply:
I went ahead and removed it.